protocol/bin/enabled-calls.json

```diff
@@ -1,1128 +1,398 @@
-    "promise": true
-  },
-  "keybase.1.teams.teamGetSubteams": {
-    "promise": true
-  },
-  "keybase.1.teams.teamIgnoreRequest": {
-    "promise": true
-  },
-  "keybase.1.teams.teamLeave": {
-    "promise": true
-  },
-  "keybase.1.teams.teamListMyAccessRequests": {
-    "promise": true
-  },
-  "keybase.1.teams.teamListRequests": {
-    "promise": true
-  },
-  "keybase.1.teams.teamListUnverified": {
-    "promise": true
-  },
-  "keybase.1.teams.teamProfileAddList": {
-    "promise": true
-  },
-  "keybase.1.teams.teamReAddMemberAfterReset": {
-    "promise": true
-  },
-  "keybase.1.teams.teamRemoveMember": {
-    "promise": true
-  },
-  "keybase.1.teams.teamRename": {
-    "promise": true
-  },
-  "keybase.1.teams.teamSetSettings": {
-    "promise": true
-  },
-  "keybase.1.teams.untrustedTeamExists": {
-    "promise": true
-  },
-  "keybase.1.teams.uploadTeamAvatar": {
-    "promise": true
-  },
-  "keybase.1.test.echo": {
-    "promise": true
-  },
-  "keybase.1.track.checkTracking": {
-    "promise": true
-  },
-  "keybase.1.track.dismissWithToken": {
-    "promise": true
-  },
-  "keybase.1.track.trackWithToken": {
-    "promise": true
-  },
-  "keybase.1.track.untrack": {
-    "promise": true
-  },
-  "keybase.1.user.blockUser": {
-    "promise": true
-  },
-  "keybase.1.user.canLogout": {
-    "promise": true
-  },
-  "keybase.1.user.dismissBlockButtons": {
-    "promise": true
-  },
-  "keybase.1.user.getUserBlocks": {
-    "promise": true
-  },
-  "keybase.1.user.interestingPeople": {
-    "promise": true
-  },
-  "keybase.1.user.listTrackersUnverified": {
-    "promise": true
-  },
-  "keybase.1.user.listTracking": {
-    "promise": true
-  },
-  "keybase.1.user.loadMySettings": {
-    "promise": true
-  },
-  "keybase.1.user.loadPassphraseState": {
-    "promise": true
-  },
-  "keybase.1.user.profileEdit": {
-    "promise": true
-  },
-  "keybase.1.user.proofSuggestions": {
-    "promise": true
-  },
-  "keybase.1.user.reportUser": {
-    "promise": true
-  },
-  "keybase.1.user.setUserBlocks": {
-    "promise": true
-  },
-  "keybase.1.user.unblockUser": {
-    "promise": true
-  },
-  "keybase.1.user.uploadUserAvatar": {
-    "promise": true
-  },
-  "keybase.1.user.userCard": {
-    "promise": true
-  },
-  "keybase.1.userSearch.getNonUserDetails": {
-    "promise": true
-  },
-  "keybase.1.userSearch.userSearch": {
-    "promise": true
-  },
-  "keybase.1.userSearch.bulkEmailOrPhoneSearch": {
-    "promise": true
-  },
-  "keybase.1.wot.wotVouch": {
-    "promise": true
-  },
-  "keybase.1.wot.wotReact": {
-    "promise": true
-  },
-  "stellar.1.local.getWalletAccountsLocal": {
-    "promise": true
-  },
-  "stellar.1.local.deleteWalletAccountLocal": {
-    "promise": true
-  },
-  "stellar.1.local.getWalletAccountSecretKeyLocal": {
-    "promise": true
-  },
-  "stellar.1.local.hasAcceptedDisclaimerLocal": {
-    "promise": true
-  }
+  "chat.1.chatUi.chatLoadGalleryHit": {"incoming": true},
```

Comment:
i removed a bunch of whitespace, removed unused calls and added custom/incoming so only gen what we use